api/Console.json

```
@@ -1480,19 +1516,22 @@
               "notes": "Alias for <code>console.error</code>"
             },
             "opera": {
-              "version_added": true
+              "version_added": "10.5"
+            },
+            "opera_android": {
+              "version_added": "11"
             },
             "safari": {
               "version_added": true
```

```suggestion
              "version_added": "3"
```

Same reasoning as for console.info.